### DISCUSSION TITLE
E282: Cannot read from "c:\users\username\appdata\local\nvim\init.vim"

### DISCUSSION THREAD

**Discussion by HiloCoder** [vscode-neovim/vscode-neovim#1852] [Q&A]
I am using VSCode-NeoVim plugin with neovim 0.9.5 on Windows 11.  I would like to use an init.lua file, which is in the directory where VSCode-NeoVim is trying to read the init.vim.   The init.lua file works fine when launching nvim.exe and nvm-qt.exe.  I don't see any setting in VSCode-NeoVim to specify that I want to use init.lua.  Does it only work with init.vim?

**Comment by theol0403** [comment]
You can omit the configuration path option, or point it directly to the Lua file.

**Reply by HiloCoder** [reply]
How?  The only path option in the plugin settings is "Neovim Init Vim Paths: Win32"  which does not work at all.  Regardless what is entered in that setting yields the same error.

**Reply by xiyaowong** [reply]
Remove this option.

**Reply by HiloCoder** [reply]
I don't use that setting because it doesn't work. It is blank. Same error.
